pysb/simulator/base.py

```diff
@@ -224,27 +234,72 @@ class SimulationResult(object):
         A set of species trajectories from a simulation. Should either be a
         list of 2D numpy arrays or a single 3D numpy array.
 
-    Attributes
```

Comment:
Where did the Attributes section go? Does this somehow get auto-populated from the `@property` method docstrings?